packages/pds-ember/addon/components/pds/external-link/stories/index.stories.js

```diff
@@ -0,0 +1,40 @@
+import ICONS from '@hashicorp/structure-icons/dist/index';
+import hbs from 'htmlbars-inline-precompile'
+import DocsPage, { TITLE } from '../docs.mdx'
+
+export default {
+  title: TITLE,
+  component: 'PdsExternalLink',
+  parameters: { docs: { page: DocsPage } },
+}
+
+export const Index = (args) => ({
+  template: hbs`
+    <Pds::ExternalLink
+      href="#"
+      @hideIcon={{hideIcon}}
+    >
+      {{#if iconType}}
+        <Pds::Icon @type={{iconType}} />
+      {{/if}}
```

Comment:
Instead of `@variant={{variantName}}`, you can use `<Pds::Icon @type={{variantName}} />` in the component block. (reduces component complexity)